mwdb/core/rate_limit.py

```diff
@@ -1,32 +1,60 @@
-import time
-
-import redis
-from flask import g
-from werkzeug.exceptions import TooManyRequests
+from flask import g, request
+from flask_limiter import Limiter
 
+from mwdb.core.capabilities import Capabilities
 from mwdb.core.config import app_config
 
 
-def rate_limit(key, duration, limit):
-    class TooManyRequestsWithRetryAfter(TooManyRequests):
-        def __init__(self, retry_after):
-            self.retry_after = retry_after
-            super().__init__(
-                f"You are too fast. Wait {retry_after} seconds before next request."
+def rate_limit_enabled():
+    if (
+        app_config.mwdb.enable_rate_limit
+        and g.auth_user is not None
+        and not g.auth_user.has_rights(Capabilities.unlimited_requests)
+    ):
+        return True
+    else:
+        return False
+
+
+limiter = Limiter(
+    key_func=lambda: f"{g.auth_user.login}-{request.method}"
+    if rate_limit_enabled()
+    else None,
```

Comment:
`Retry-After` header is used by mwdblib and should be enabled
```suggestion
    else None,
    headers_enabled=True,
```